index.html

```diff
@@ -1,168 +1,309 @@
 ---
 layout: default
 ---
-<section class="row">
 
   <!-- Hero banner -->
-  <div class="twelve-col">
+  <div class="hero">
 
-    <span class="logo">
-      Cloud Init
-    </span>
+    <section class="row">
 
-    <h1>The standard for customising cloud instances</h1>
+      <div class="wrapper">
 
-    <p>cloud-init allows you to launch an instance which will integrate with a configuration management solution of your choice.</p>
+        <div class="twelve-col">
 
-    <div class="six-col">
-      <a href="https://www.youtube.com/watch?v=-zL3BdbKyGY" class="button--primary">
-        Watch tutorial
-      </a>
-    </div>
+          <div class="logo">
+            <img class="logo__img" src="https://assets.ubuntu.com/v1/4fd55d51-picto-corepower-orange.svg" alt="Cloud Init logo" />
+            <span class="logo__txt">Cloud Init</span>
+          </div>
+
+          <h1 class="hero__heading">The standard for customising cloud instances</h1>
+
+          <p class="hero__summary">Cloud images are operating system templates
+            and every instance starts out as an identical clone of every other
+            instance. It is the user data that gives every cloud instance its
+            personality and cloud-init is the tool applies user data to your
+            instances automatically.
+          </p>
+
+          <div class="six-col">
+            <a href="https://www.youtube.com/watch?v=-zL3BdbKyGY" class="hero__cta button--primary">
+              Watch tutorial
+            </a>
+          </div>
+        </div>
+      </div>
+    </section>
   </div>
-</section>
 
-<!-- Main selling points -->
-<section class="row">
+<div class="wrapper">
 
-  <div class="twelve-col">
+  <!-- Main selling points -->
+  <section class="row">
```

Comment:
@anthonydillon - The wrapper is required to give a max-width and a center column to all content other than the hero banner. See screenshot.

![screen shot 2016-01-27 at 11 31 12](https://cloud.githubusercontent.com/assets/505570/12612348/f2252e74-c4e9-11e5-825f-fdaae70627f4.png)